packages/react-components/react-migration-v8-v9/library/tsconfig.json

```diff
@@ -8,8 +8,7 @@
     "importHelpers": true,
     "jsx": "react",
     "noUnusedLocals": true,
-    "preserveConstEnums": true,
-    "paths": {}
```

Comment:
this was inccorectly added during split migration - undoing to fix TS intellisense